packages/library/src/providers/ModalProvider/ModalProvider.tsx

```diff
@@ -19,6 +19,9 @@ import { useBreakpoint } from '@deriv/quill-design';
 type TModalState = {
     // Add your modal states here
     modalType?: string; // Just an example
+    marketType?: string;
```

Comment:
```suggestion
    marketType?: TMarketTypes.All;
    platform: TPlatforms,All;
```

Reply:
@mahdiyeh-deriv moved it to useCFDContext